packages/Panel/src/components/Group/Group.js

```diff
@@ -30,10 +30,6 @@ export default function Group(props) {
 
   const offsetScroll = useOffsetScroll(offsetY);
 
-  if (sidePanels.filter(sidePanel => !sidePanel.props.isOpen).length === sidePanels.length) {
-    return null;
-  }
-
```

Comment:
Removing this as currently the parent panel doesn't animate close when rendered in a `Panel.Group`. This also means it doens't trigger an `onAfterClose` callback, which we require to change the application url in Workflows UI

Can be seen [here](http://storybooks.highbond-s3.com/paprika/master/?path=/story/messaging-panel-examples--group-panel)